app/views/sephora_style_guide/home/_pagination.slim

```diff
@@ -83,3 +83,26 @@ article#pagination
       a.page 11
       a.page.current 12
       a.page.next
+
+  pre
+    code.language-pug
+      |
+        /- For between pages (Top)
+        nav.pagination
+          a.page.prev
+          a.page 7
+          a.page.current 8
+          a.page 9
+          a.page.next
+
+        /- For between pages (Bottom)
+        nav.pagination
+          a.page.prev
+          a.page 1
+          a.page.continue ...
+          a.page.current 7
+          a.page 8
+          a.page 9
+          a.page.continue ...
+          a.page 12
+          a.page.next
```

Comment:
Missing final new line. If you are not aware, EditrorConfig should fix these automatically for you.

Reply:
Somehow it was only working for the main repo.. have it enabled now for the style guide.